### ISSUE TITLE
Project State

### ISSUE THREAD

**Issue by albsch** [nibdo/bloben-app#124]
Hi, 

I'm wondering whether the project is abandoned or not. All projects seem dead since January, and there is no further activity @nibdo . Are there any plans to continue maintaining the project?

If there is no answer, @community:

Is there any interest in continuing the project in an open source organization?

**Comment by Opisek** [comment]
The website appears to be down as well.

**Comment by Opisek** [comment]
@albsch Tag me if you end up creating an organization to keep a fork running. Otherwise, I'll privately start something similar from scratch.

**Comment by albsch** [comment]
As of now I have no intention of creating an organization, so go ahead @Opisek :+1:

**Comment by tiritibambix** [comment]
I'm sad such a nice project is not under developpment anymore.
@Opisek , did you end up working on something ?

**Comment by Opisek** [comment]
Yeah, I'm working on [Luna](https://github.com/Opisek/luna). Note that progress is slow since I'm currently busy with university, but I expect it to expedite during my vacation starting around august.

**Comment by tiritibambix** [comment]
> Yeah, I'm working on [Luna](https://github.com/Opisek/luna). Note that progress is slow since I'm currently busy with university, but I expect it to expedite during my vacation starting around august.

Can't wait to see your work. Just stared Luna.
Are you planning on supporting docker ?
In the meantime, good luck with university

**Comment by Opisek** [comment]
Thanks for the star. Yes, Docker will be the default deployement method. The infrastructure will likely be composed of the frontend, the backend, and a PostgreSQL database, all in individual containers. I will include Docker build files and an example docker-compose once I'm that far into development.

**Comment by tiritibambix** [comment]
> Thanks for the star. Yes, Docker will be the default deployement method. The infrastructure will likely be composed of the frontend, the backend, and a PostgreSQL database, all in individual containers. I will include Docker build files and an example docker-compose once I'm that far into development.

This is great news. Thanks is for your answer.
